### PR TITLE
Release 0.4

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog {#Changelog}
 
 # git master {master)
 
+# Release 0.4.0 (03-10-2016) {#release_004000}
```
Comment (hernando) [inline doc/Changelog.md]: The date format is dd-mm-yyyy

**Comment by hernando** [comment]
Needs rebasing on top of https://github.com/BlueBrain/Monsteer/pull/19

**Comment by bilgili** [comment]
Also, waiting for Brion to be released and it has an open pull request.

**Comment by bilgili** [comment]
#20 handles this pr

**Pull request closed without merging**